generators/csharp/sdk/src/SdkGeneratorContext.ts

```diff
@@ -111,6 +122,31 @@ export class SdkGeneratorContext extends AbstractCsharpGeneratorContext<SdkCusto
         return `${this.getComputedClientName()}Client`;
     }
 
+    public getBaseExceptionClassReference(): csharp.ClassReference {
+        return csharp.classReference({
+            name: this.getExceptionPrefix() + "Exception",
+            namespace: this.getCoreNamespace()
+        });
+    }
+
+    public getBaseApiExceptionClassReference(): csharp.ClassReference {
+        return csharp.classReference({
+            name: this.getExceptionPrefix() + "ApiException",
+            namespace: this.getCoreNamespace()
+        });
+    }
+
+    public getExceptionClassReference(declaredErrorName: DeclaredErrorName): csharp.ClassReference {
+        return csharp.classReference({
+            name: this.getPascalCaseSafeName(declaredErrorName.name),
+            namespace: this.getNamespaceFromFernFilepath(declaredErrorName.fernFilepath)
```

Comment:
@dsinghvi i know we very briefly discussed if we should have the `explicit-namespaces` config apply here and landed on `no`

fysa i've reversed on that and think it's easiest and most consistent (and not too consequential) to apply the config